### DISCUSSION TITLE
Blog post: "Why We Ditched RabbitMQ And Replaced It With A Postgres Queue"

### DISCUSSION THREAD

**Discussion by jrochkind** [bensheldon/good_job#925] [General]
Saw this one on Hacker News today, and seemed of interest to good_job developers and enthusiasts! While the specific context is not ours, just fellow people using postgres as a queue. 

https://www.prequel.co/blog/sql-maxis-why-we-ditched-rabbitmq-and-replaced-it-with-a-postgres-queue

https://news.ycombinator.com/item?id=35526846

**Comment by jrochkind** [comment]
Oh and hey, someone already mentioned good_job in the HN comments, with someone else asking for more info! https://news.ycombinator.com/item?id=35528230

**Reply by bensheldon** [reply]
AWESOME 🎉 thank you for the ping and sharing this. This is fantastic!!!
